lib/vsc/mympirun/pmi/__init__.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2009-2019 Ghent University
```

Comment:
Actually, are you not supposed to have the start year be the year when the file was first created?

Reply:
yep, will fix

Reply:
i'll not fix it. this is a copy of the original __init_.py afterall
the actual new files with new copyrightable material have proper header :stuck_out_tongue:

Reply:
ic. but it is still a new file.